backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/PredicatesHandler.java

```diff
@@ -153,6 +155,84 @@
     return InternalFuture.completedInternalFuture(queryContext);
   }
 
+  private InternalFuture<QueryFilterContext> processAttributePredicate(
+      long index, KeyValueQuery predicate, String name, String fieldType) {
+    try {
+      final var value = predicate.getValue();
+      var operator = predicate.getOperator();
+      //  We will manage this operator at bottom of method based on `NE` and `NOT_CONTAIN` using
+      // `IN` OR `NOT IN` query
+      if (operator.equals(OperatorEnum.Operator.NOT_CONTAIN)) {
+        operator = OperatorEnum.Operator.CONTAIN;
+      } else if (operator.equals(OperatorEnum.Operator.NE)) {
+        operator = OperatorEnum.Operator.EQ;
+      }
```

Comment:
Change as discussed

Reply:
i have changed it in https://github.com/VertaAI/modeldb/pull/2179/files so i will change here after merging it because both are used same operator function.

Reply:
Sure

Reply:
Removed this code as per suggestions